helm-chart-sources/pulsar/ci-archive/gcp-storage-no-test.yaml

```diff
@@ -32,6 +32,7 @@ storageOffload:
   driver: google-cloud-storage
   gcsServiceAccountSecret: pulsar-gcp-sa-secret # pragma: allowlist secret
   gcsServiceAccountJsonFile: account-223201-f12856532197.json
+  gcsServiceAccountJsonFileContent: <must be passed from env variable in Helm command line> # this should be a base64-encoded string $(cat <gcsServiceAccountJsonFile> | base64)
```

Comment:
@MMirelli - are you familiar with helm templates? Instead of expecting users to base64 encode the file content, we could take in the file content and then use a template to encode it for users. There is an example of using a helper template here: https://helm.sh/docs/howto/charts_tips_and_tricks/#creating-image-pull-secrets, however, we could also encode it in the template file.

Reply:
We spoke offline and decided that for now, we will expect users to base64 encode this value. There is documentation showing how to do so.